packages/trpc/server/routers/viewer/bookings/confirm.handler.ts

```diff
@@ -113,6 +116,25 @@ export const confirmHandler = async ({ ctx, input }: ConfirmOptions) => {
     throw new TRPCError({ code: "BAD_REQUEST", message: "Booking already confirmed" });
   }
 
+  // Check for user's slot availability for current booking start and end time
+  if (confirmed) {
+    const slotsAvailable = await getAvailableSlots({
+      startTime: booking?.startTime.toISOString(),
+      endTime: booking?.endTime.toISOString(),
+      eventTypeId: booking?.eventType?.id,
+      eventTypeSlug: booking?.eventType?.slug,
+      timeZone: user?.timeZone,
+      usernameList: getUsernameList(user?.username ?? ""),
+      isTeamEvent: false,
```

Comment:
why did you pass false here ?

Reply:
Yes, updated.
The result would be same for this scenario, even if we default to false, as here we are concerned about single user's or admin's slots only. But anyway updated to be in consistent.